### ISSUE TITLE
[Markup and meta data] HTML markup isn't valid. (09.01.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#68258]
#### VFS team name
Authenticated Experience

#### VFS product name
My VA

#### VFS feature name
Dot indicator

#### Point of Contact/Reviewers
Ryan Smith - @rsmithadhoc - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
HTML markup isn't valid.

### Issue Details
`aria-label` is only supported on interactive elements. This does not get read out by a screen reader.
```html
<i aria-label="Unread messages. Go to your inbox." class="fas fa-xs fa-stack-1x vads-u-height--full fa-circle vads-u-color--secondary-dark"></i>
```

### VA.gov Experience Standard
[Category Number 09, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A

### Platform Recommendation
This `aria-label` can be removed. There is already a label on the link element which accommodates screen readers. On the icon, add `aria-hidden="true"` to hide it from assistive technologies, since the link will have the semantic label to inform the user of unread messages.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by ajakabcin** [comment]
Auth experience My VA team will address this in #68307 and #68309

**Comment by aagosto90** [comment]
Hi @rsmithadhoc , thank you for the feedback! Forgive my beginner accessibility questions. I feel like I always struggle with aria labels. We want the screen reader to announce "Unread messages" and then the link title, "Go to your inbox." How do you suggest we code that?

**Comment by rsmithadhoc** [comment]
> Hi @rsmithadhoc , thank you for the feedback! Forgive my beginner accessibility questions. I feel like I always struggle with aria labels. We want the screen reader to announce "Unread messages" and then the link title, "Go to your inbox." How do you suggest we code that?

@aagosto90 Currently, I believe it is coded something like:
```html
<a href="#" aria-label="Unread messages">
    Go to your inbox <i aria-label="Unread messages. Go to your inbox." class="fas fa-circle"></i>
</a>
```
For a screen reader, `aria-label` will override the link text. It will only read the first`aria-label` on the link, "Unread messages", not the link text itself. 

The second `aria-label` is extraneous because `<i>` is not an interactive element. Nothing will be read out for this. However, without an ARIA attribute, I think accessibility tools might report this as an issue of the icon not having alternative text. In this case, we want to treat it as presentational only. Because we have an `aria-label` on the link itself to inform screen reader users of unread notifications, an `aria-label` on the icon is redundant, so we can safely hide that from screen reader.

So I think this is how it should be coded to address both of the accessibility issues reported:
```html
<a href="#" aria-label="Unread messages. Go to your inbox">
    Go to your inbox <i aria-hidden="true" class="fas fa-circle"></i>
</a>
```
And when there are no unread messages, it can be a standard link with no ARIA attributes:
```html
<a href="#">
    Go to your inbox
</a>
```

Can you let me know if that helps?

**Comment by aagosto90** [comment]
I think that I get it @rsmithadhoc but I'm going to reiterate in plain non-coding person language to make sure that I do and then pass on your guidance to our engineers. Basically we need to include the link text in the aria label because that's exactly what will be read out on a screen reader. And then we don't need an aria label when there are no unread messages. Does that sound right?

**Comment by rsmithadhoc** [comment]
> I think that I get it @rsmithadhoc but I'm going to reiterate in plain non-coding person language to make sure that I do and then pass on your guidance to our engineers. Basically we need to include the link text in the aria label because that's exactly what will be read out on a screen reader. And then we don't need an aria label when there are no unread messages. Does that sound right?

Yup, that sounds correct.

**Comment by ajakabcin** [comment]
Resolved - see confirmation [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/68309#issuecomment-1796175769)
